### PR TITLE
Kin front end

### PR DESCRIPTION
**Pull request by Joaquin808**
Just need to pull

### CODE REVIEW

**Review by lamarhinchen — APPROVED** [review]

File: JavaAPI/pom.xml [hunk 1]
```diff
@@ -136,6 +136,11 @@
             <artifactId>log4j-core</artifactId>
             <version>2.14.1</version>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
```
Comment (marcopol07) [inline JavaAPI/pom.xml:140]: We do not need slf4j, it is for logging: we are using log4j instead so make sure you have that dependency

Comment (marcopol07) [reply JavaAPI/pom.xml:140]: So, remove this dependency

File: JavaAPI/src/main/java/app/App.java [hunk 2]
```diff
@@ -22,13 +22,15 @@ private static void establishRoutes(Javalin app) {
 
         //EndPoints
         AssessmentController ac = new AssessmentController(as);
```
Comment (marcopol07) [inline JavaAPI/src/main/java/app/App.java:24]: This file is out of date, remove it from commit

File: JavaAPI/pom.xml (hunk 1 above)
Comment (JGay90) [inline JavaAPI/pom.xml:140]: log4j needs that dependency Marc

Comment (marcopol07) [reply JavaAPI/pom.xml:140]: Yes, I have become aware. Thanks!

**Review by JGay90 — APPROVED** [review]

**Pull request merged**